src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -1,13 +1,29 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Excavator
- * Do not remove no-args constructor
- */
 public class Excavator extends Machine {
+
+    private double bucketCapacity;
+    private int armLength;
```

Comment:
The field `armLength` should have an access modifier to encapsulate the field properly. Consider using a private modifier if there's no need to expose it directly.